string/breaking_bad.py

```diff
@@ -35,8 +49,36 @@ def match_symbol(chemicals, symbols):
 
     return combined
 
+print(match_symbol(chemicals, symbols))
+
+
+
+"""
+An improved version of above, get right output.
+
+symbols = ['I', 'Am', 'cro', 'Na', 'le', 'abc', 'o']
+words = ['Amazon', 'Microsoft', 'Google']
```

Comment:
Pls maintain common list names throughout the code. I could see the 'chemicals' denoting the name of the list in match_symbols()